crates/linear_algebra/src/framed.rs

```diff
@@ -11,6 +11,7 @@ use std::{
 };
 
 #[derive(Debug)]
+#[repr(transparent)]
```

Comment:
leave a comment here, explaining why it is here to ensure that this is not removed in the future